tests/integration/components/justa-table-test.js

```diff
@@ -370,12 +371,16 @@ test('recalculates fixed column width', function(assert) {
 
     {{/justa-table}}
   `);
+
   return wait().then(() => {
+    const columnsElem = getNode(this).querySelector('.fixed-table-columns-wrapper');
+
     // this currently includes 1 px border from the appearance css. not ideal.
-    assert.equal(this.$('.fixed-table-columns-wrapper').width(), 151);
+    assert.equal(parseInt(window.getComputedStyle(columnsElem).width), 151);
```

Comment:
👍  on moving to this format. I imagine anywhere we get the width should move to this style, but baby steps. 😄